source/jormungandr/tests/tests_mechanism.py

```diff
@@ -367,6 +367,10 @@ def is_valid_journey_response(self, response, query_str):
                 assert j['links'][0]['type'] == 'journeys'
                 assert '/journeys?' in j['links'][0]['href']
                 assert 'allowed_id%5B%5D=' in j['links'][0]['href']
+                if 'first_secion_mode%5B%5D=' in j['links'][0]['href']:
+                    assert j['links'][0]['href'].count('first_secion_mode%5B%5D=') == 1
```

Comment:
assert j['links'][0]['href'].count('first_secion_mode%5B%5D=') <= 1 no ?

and can you add a comment ?